.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-Xmx2048m
```

Comment:
sure set a value - but not 2GB...  size this appropriately - if we really do need huge amounts of RAM then one of the maven plugins is probably miss-behaving / inefficient and we should look at fixing that and keeping this set to a meaningful value.